projects/angular/src/popover/signpost/signpost-content.ts

```diff
@@ -40,7 +40,7 @@ const POSITIONS: string[] = [
       <div class="signpost-content-header">
         <button
           type="button"
-          [attr.aria-label]="commonStrings.keys.signpostClose"
+          [attr.aria-label]="signpostCloseAriaLabel || commonStrings.keys.signpostClose"
```

Comment:
```suggestion
          [attr.aria-label]="signpostCloseAriaLabel"
```

Reply:
The current implementation is better so that we default to the common string if a falsy value is bound to the input. This is a pattern we use in several places, and I think this one should be the same.

Reply:
@dtsanevmw They want to customise aria label with name of trigger element, so current implementation is better and we are using same pattern in other places also.

Reply:
We have kind of mixed approaches and while If someone uses the input to bind to something that's 'falsy' is their responsibility I'm okey keeping it like you did it initially @spadidapelli.